### PR TITLE
test: improve snapshots of watermark

### PR DESCRIPTION
**Pull request by colincclark**
- added README
- improved snapshots
- added knobs to story for watermark sizes

### CODE REVIEW

File: packages/watermark/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Watermark
+
+A Times standard image (native) or svg (web) that acts as a placeholder image
+wherever required. The Watermark component can act as a placeholder for failed
+images, or as a loading state. It requires height and width props for layout.
```
Comment (craigbilner) [inline packages/watermark/README.md]: we need to automate this along the lines of TypeScript -> prop types -> doc gen -> README

File: packages/watermark/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+The code can be formatted and linted in accordance with the agreed standards.
+
+```
+yarn fmt
+yarn lint
+```
+
+## Testing
+
+This package uses [yarn](https://yarnpkg.com) (latest) to run unit tests on each
```
Comment (craigbilner) [inline packages/watermark/README.md]: think you can use the `npm` client too?

File: packages/watermark/__tests__/android/__snapshots__/watermark.android.test.js.snap [hunk 3]
```diff
@@ -1,9 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Watermark tests on android renders correctly 1`] = `
+exports[`1. watermark 1`] = `
```
Comment (craigbilner) [inline packages/watermark/__tests__/android/__snapshots__/watermark.android.test.js.snap]: would be good to test the correct src is used?

File: packages/watermark/__tests__/shared.js [hunk 4]
```diff
@@ -1,14 +1,31 @@
...
-      .create(<Watermark height={250} width={300} />)
-      .toJSON();
+export default () => {
+  addSerializers(
+    expect,
+    enzymeTreeSerializer(),
+    compose(
+      print,
+      minimaliseTransform(
+        (value, key) => key === "d" || key === "id" || key === "source"
```
Comment (craigbilner) [inline packages/watermark/__tests__/shared.js]: I would test `id` and `d` myself. Would be good to see that rather odd id of "Page-1" and if `d` had changed in anyway?

File: packages/watermark/README.md (hunk 2 above)
Comment (colincclark) [reply packages/watermark/README.md]: removed `yarn` link - would just be more maintenance re: yarn version, etc

File: packages/watermark/__tests__/shared.js (hunk 4 above)
Comment (colincclark) [reply packages/watermark/__tests__/shared.js]: done and amended `id` to something more sensible

File: packages/watermark/__tests__/android/__snapshots__/watermark.android.test.js.snap (hunk 3 above)
Comment (colincclark) [reply packages/watermark/__tests__/android/__snapshots__/watermark.android.test.js.snap]: done - mocked the asset, but this is not ideal so also created https://github.com/newsuk/times-components/issues/1102

**Pull request merged**